spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/cloudfoundry/reactive/ReactiveTokenValidator.java

```diff
@@ -43,7 +44,7 @@
 
 	private final ReactiveCloudFoundrySecurityService securityService;
 
-	private Map<String, String> cachedTokenKeys = new ConcurrentHashMap<>();
+	private volatile Map<String, String> cachedTokenKeys = Collections.emptyMap();
```

Comment:
Based on [this comment](https://github.com/spring-projects/spring-boot/pull/11233#discussion_r154333769), I think the field should be declared as a `ConcurrentMap` to avoid accidental use of a plain `Map`.